.github/CODEOWNERS

```diff
@@ -9,7 +9,7 @@
 /internal/                  @FerretDB/leads
 
 /internal/backends/         @AlekSi
-/internal/handlers/hana/    @AlekSi
+/internal/backends/hana/    @AlekSi
```

Comment:
I kept this to make it explicit that hana backend has different codeowners rules (even after we set codeowners to the team). Alternatively, I can remove this line completely.

Reply:
It is actually not possible to add them there, see (private) FerretDB/ferreteria#45